### ISSUE TITLE
Default value of produceGeneralizedRdf should be false

### ISSUE THREAD

**Issue by kal** [w3c/json-ld-api#503]
The definition of the [JsonLdOptions Type](https://www.w3.org/TR/json-ld11-api/#the-jsonldoptions-type) specifies that the default value for produceGeneralizedRdf is `true`. However, there are tests that fail if this is the case (e.g. the toRdf test case e068 has an expected output that excludes the statement with a blank node property), and I note that the test manifests explicitly set the option to `true` when it is required for the test. That, plus the fact that this option is deprecated leads me to suspect that the definition of the options type is perhaps incorrect and that the default value for this property should be `false`.

**Comment by gkellogg** [comment]
Yes, the default should be `false`, which is consistent with the warning that it may go away in the future.

**Comment by gkellogg** [comment]
Summary:

The default value of the [produceGeneralizedRdf](https://www.w3.org/TR/json-ld11-api/#dom-jsonldoptions-producegeneralizedrdf) option should be `false`, not `true`.
